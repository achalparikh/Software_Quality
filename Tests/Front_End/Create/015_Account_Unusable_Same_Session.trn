10                      00000 00000000 A 
05 Jeff Doe             00013 00000.00   
00                      00000 00000000   