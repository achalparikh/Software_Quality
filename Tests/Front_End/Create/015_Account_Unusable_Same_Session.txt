login
admin 
create
Jeff Doe
0.00
deposit
00013
500.00
logout