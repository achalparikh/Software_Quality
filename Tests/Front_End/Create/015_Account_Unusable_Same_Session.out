Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as an admin
Enter a command:
Enter an account holder:
Enter a balance for the new account:
Account for "Jeff Doe" with account number 00013 created successfully
Enter a command:
Enter an account holder:
Enter an account number belonging to the account holder:
Error, account 00013 cannot perform any transactions for the remainder of the day
Enter a command:
You are now logged out